daemon/sdnotify.go

```diff
@@ -2,30 +2,33 @@
 package daemon
 
 import (
-	"errors"
 	"net"
 	"os"
 )
 
-var SdNotifyNoSocket = errors.New("No socket")
-
 // SdNotify sends a message to the init daemon. It is common to ignore the error.
```

Comment:
@s-urbaniak while the (bool, error) signature is not nice, @alban's concern is that this error is typically ignored (following C practice). You are arguing for the status quo, just removing the "ignore" bits from the docstring. Either is fine with me.